.github/workflows/test-phpunit.yml

```diff
@@ -27,6 +27,9 @@ on:
       - phpunit.xml.dist
       - .github/workflows/test-phpunit.yml
 
+env:
+  COVERAGE_PHP_VERSION: '8.0'
```

Comment:
Why not 8.1?

Reply:
If you all agree, I can bump php version for coverage in separate PR.

Reply:
8.1 is fine. It is the most used version in 8.x now.

Reply:
Let's do it.